sos/report/plugins/libvirt.py

```diff
@@ -42,6 +42,7 @@ def setup(self):
             "/etc/libvirt/storage/autostart/*.xml",
             "/etc/libvirt/qemu-lockd.conf",
             "/etc/libvirt/virtlockd.conf",
+            "/etc/libvirt/virtlogd.conf"
```

Comment:
you'll probably get an alert from one of the tests for the missing comma here

Reply:
whoops, fixed now.